merlin/models/tf/inputs/base.py

```diff
@@ -44,7 +44,6 @@ def InputBlock(
     post: Optional[BlockType] = None,
     aggregation: Optional[TabularAggregationType] = None,
     seq: bool = False,
-    max_seq_length: Optional[int] = None,
```

Comment:
This argument is removed because it doesn't seem to be used anywhere in the function.